index.html

```diff
@@ -85,5 +85,29 @@
     of code. If it needs to be a bit bigger, it will be.
   </li> 
 </ol> 
+
+<p>
+In addition to these stated goals, we try our best to uphold the following
+values when considering contributions to the project:
+</p>
+
+<ul>
+  <li>
+    Never break configuration files or existing workflows. Breaking changes
+    require a major version bump (v4 → v5).
```

Comment:
Should we mention semantic versioning? Do we even strive to achieve that? If so, I think the breaking binding changes, for example, would have warranted an immediate v5 release. I think it's worth considering how such a situation should be handle in the future (not necessarily written down, it just popped into my head).